libyul/backends/evm/EVMDialect.cpp

```diff
@@ -173,6 +173,8 @@ std::set<YulName> createReservedIdentifiers(langutil::EVMVersion _evmVersion)
 		"datacopy"_yulname,
 		"setimmutable"_yulname,
 		"loadimmutable"_yulname,
+		"auxdataloadn"_yulname,
```

Comment:
Just wanted to note that it would be best to put left-overs from previous PRs like this in a separate commit. That's really a bugfix.

That commit could also include the other similar change, i.e. moving the `auxdataloadn` test to `eof/`.

Reply:
Yes. I will move it.